### PR TITLE
[it] Insert API Gateway term

### PR DESCRIPTION
**Pull request by akelity**
Signed-off-by: Michel Murabito <michel.murabito@mia-platform.eu>

### CODE REVIEW

File: content/it/api-gateway.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: API Gateway
+status: Completed
+category: tecnologia
+tags: ["networking", "", ""]
```
Comment (jihoon-seo) [inline content/it/api-gateway.md:5]: Which one would be great here, between:
- "networking"
- the Italian term of "networking"

Preview: https://deploy-preview-1092--cncfglossary.netlify.app/it/api-gateway/
image

@akelity PTAL!

---

For anyone's information:
the "Tags" page: https://deploy-preview-1092--cncfglossary.netlify.app/it/tags/
image

File: content/it/api-gateway.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+## Cos'è  
+Un [API](/it/application-programming-interface/) Gateway è uno strumento che aggrega API applicative rendendole disponibili in un'unica posizione. Questo consente alle organizzazioni di spostare le funzioni chiave, come l'autenticazione e l'autorizzazione o la limitazione del numero di richieste tra le applicazioni, in una posizione gestita centralmente. Un API Gateway fornisce inoltre un'unica interfaccia comune per i consumatori di API (spesso esterni).
+
+
+## Quali problematiche risolve
```
Comment (fsbaraglia) [inline content/it/api-gateway.md:12]:
```suggestion
## Quali problematiche affronta  
```

**Review by fsbaraglia — CHANGES_REQUESTED** [review]

**Review by fsbaraglia — APPROVED** [review]

File: content/it/api-gateway.md (hunk 1 above)
Comment (akelity) [reply content/it/api-gateway.md:5]: Networking (in the technical field) is a term commonly used in Italy

File: content/it/api-gateway.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: API Gateway
+status: Completed
+category: tecnologia
+tags: ["networking", "", ""]
+---
+
+## Cos'è  
+Un [API](/it/application-programming-interface/) Gateway è uno strumento che aggrega API applicative rendendole disponibili in un'unica posizione. Questo consente alle organizzazioni di spostare le funzioni chiave, come l'autenticazione e l'autorizzazione o la limitazione del numero di richieste tra le applicazioni, in una posizione gestita centralmente. Un API Gateway fornisce inoltre un'unica interfaccia comune per i consumatori di API (spesso esterni).
```
Comment (sistella) [inline content/it/api-gateway.md:9]: ho visto che in inglese è effettivamente così, ma io questo link lo sposterei al chiaro riferimento alle api
```suggestion
Un API Gateway è uno strumento che aggrega [API](/it/application-programming-interface/) applicative rendendole disponibili in un'unica posizione. Questo consente alle organizzazioni di spostare le funzioni chiave, come l'autenticazione e l'autorizzazione o la limitazione del numero di richieste tra le applicazioni, in una posizione gestita centralmente. Un API Gateway fornisce inoltre un'unica interfaccia comune per i consumatori di API (spesso esterni).
```

File: content/it/api-gateway.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+
+## Quali problematiche affronta  
+Se stai mettendo le API a disposizione di consumatori esterni, ti consigliamo di avere un unico punto di ingresso per gestire e controllare tutti gli accessi. Inoltre, se devi applicare funzionalità aggiuntive a tali interazioni, un API Gateway ti consente di applicarle in modo uniforme a tutto il traffico senza richiedere modifiche al codice dell'app.
```
Comment (sistella) [inline content/it/api-gateway.md:13]:
```suggestion
Se stai mettendo le API a disposizione di consumatori esterni, vorresti avere un unico punto di ingresso per gestire e controllare tutti gli accessi. Inoltre, se devi aggiungere funzionalità a tali interazioni, un API Gateway ti consente di applicarle in modo uniforme a tutto il traffico senza richiedere modifiche al codice dell'app.
```

File: content/it/api-gateway.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+
+## In che modo aiuta  
+Fornendo un unico punto di accesso per le varie API in un'applicazione, gli API Gateway rendono più facile per le organizzazioni applicare logiche di business o di sicurezza trasversali in una posizione centrale. Consentono inoltre ai consumatori dell’applicazione di rivolgersi a un unico indirizzo per tutte le loro esigenze. Un API Gateway può semplificare problemi operativi come sicurezza e [osservabilità](/it/observability/) fornendo un unico punto di accesso per le richieste a tutti i servizi Web in un sistema. Poiché tutte le richieste fluiscono attraverso l’API Gateway, è possibile in un'unica posizione aggiungere funzionalità come la raccolta di metriche, la rate-limit e l'autorizzazione.
```
Comment (sistella) [inline content/it/api-gateway.md:17]:
```suggestion
Fornendo un unico punto di accesso per le varie API in un'applicazione, gli API Gateway rendono più facile per le organizzazioni applicare logiche di business o di sicurezza trasversali in una posizione centrale. Consentono inoltre ai consumatori dell’applicazione di rivolgersi a un unico indirizzo per tutte le loro esigenze. Un API Gateway può semplificare problemi operativi come sicurezza e [osservabilità](/it/observability/) fornendo un unico punto di accesso per le richieste a tutti i servizi Web in un sistema. Poiché tutte le richieste fluiscono attraverso l’API Gateway, è possibile in un'unica posizione aggiungere funzionalità come la raccolta di metriche, i limiti di utilizzo e l'autorizzazione.
```

**Review by sistella — CHANGES_REQUESTED** [review]

**Review by sistella — APPROVED** [review]

File: content/it/api-gateway.md (hunk 3 above)
Comment (akelity) [reply content/it/api-gateway.md]: Si, correttissimo.
Grazie!

**Comment by fsbaraglia** [comment]
bravo @akelity  :D

**Pull request merged**